docs/settings-gen/source/kibana-alert-action-settings.asciidoc

```diff
@@ -0,0 +1,1033 @@
+
+
+IMPORTANT: <<pre-configured-connectors,Preconfigured connectors>> are not affected by this setting.
+====
+
+.`xpack.actions.microsoftExchangeUrl`
+[%collapsible]
+====
+The URL for the Microsoft Azure Active Directory endpoint to use for MS Exchange email authentication.
+
+Default: `https://login.microsoftonline.com` +
+====
+
+.`xpack.actions.microsoftGraphApiUrl`
+[%collapsible]
+====
+The URL for the Microsoft Graph API endpoint to use for MS Exchange email authentication.
+
+Default: `https://graph.microsoft.com/v1.0` +
+====
+
+.`xpack.actions.microsoftGraphApiScope`
+[%collapsible]
+====
+The URL for the Microsoft Graph API scope endpoint to use for MS Exchange email authentication.
+
+Default: `https://graph.microsoft.com/.default` +
+====
+
+.`xpack.actions.proxyUrl` {ess-icon}
+[%collapsible]
+====
+Specifies the proxy URL to use, if using a proxy for actions. By default, no proxy is used.
+
+Proxies may be used to proxy http or https requests through a proxy using the http or https protocol.  Kibana only uses proxies in "CONNECT" mode (sometimes referred to as "tunneling" TCP mode, compared to HTTP mode).  That is, Kibana will always make requests through a proxy using the HTTP `CONNECT` method.
+
+If your proxy is using the https protocol (vs the http protocol), the setting `xpack.actions.ssl.proxyVerificationMode: none` will likely be needed, unless your proxy's certificates are signed using a publicly available certificate authority.
+
+There is currently no support for using basic authentication with a proxy (authentication for the proxy itself, not the URL being requested through the proxy).
+
+====
+
+.`xpack.actions.proxyBypassHosts` {ess-icon}
+[%collapsible]
+====
+Specifies hostnames which should not use the proxy, if using a proxy for actions. The value is an array of hostnames as strings.
+
+By default, all hosts will use the proxy, but if an action's hostname is in this list, the proxy will not be used.  The settings `xpack.actions.proxyBypassHosts` and `xpack.actions.proxyOnlyHosts` cannot be used at the same time.
+
+
+
+include::../embeds/example-xpack-actions-proxybypasshosts.asciidoc[]
+====
+
+.`xpack.actions.proxyOnlyHosts` {ess-icon}
+[%collapsible]
+====
+Specifies hostnames which should only use the proxy, if using a proxy for actions. The value is an array of hostnames as strings.
+
+By default, no hosts will use the proxy, but if an action's hostname is in this list, the proxy will be used.  The settings `xpack.actions.proxyBypassHosts` and `xpack.actions.proxyOnlyHosts` cannot be used at the same time.
+
+
+
+include::../embeds/example-xpack-actions-proxyonlyhosts.asciidoc[]
+====
+
+.`xpack.actions.proxyHeaders` {ess-icon}
+[%collapsible]
+====
+Specifies HTTP headers for the proxy, if using a proxy for actions.
+
+Default: `HASH(0x12bb51a88)` +
```

Comment:
This should be the following:

```
Default: {}
```

But it renders as the following:

<img width="233" alt="Screenshot 2024-11-08 at 2 49 38 PM" src="https://github.com/user-attachments/assets/aaa09330-a299-4b87-b992-7b24f12b960a">

Reply:
Another nice catch. Fixed in [fa1fe32](https://github.com/elastic/kibana/pull/191787/commits/fa1fe3211eecf3168dacd48bce888eb500c7537d)